packages/react-jss/src/utils/sheets-meta.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import type {StyleSheet} from 'jss'
+import type {StaticStyles} from '../types'
+
+interface SheetMeta {
+  styles: StaticStyles;
+  dynamicStyles: StaticStyles;
+  dynamicRuleCounter: number;
+}
+
+const sheetsMeta = new WeakMap<StyleSheet, SheetMeta>()
+
+function getMetaForSheet(sheet: StyleSheet) {
```

Comment:
I would add return types always

Reply:
Flow will infer them correctly. I don't think we necessarily need them.